packages/agents/cartographer/poller/README.md

```diff
@@ -31,16 +34,51 @@
 }
 ```
 
-- Run poller:
+## Run poller:
 
-```sh
+<!-- ```sh
 yarn workspace @connext/cartographer-poller start
 ```
 
 If you'd like to run a different poller, you can specify which one you'd
 like to run by setting the `SERVICE` env var, e.g.L
 
-SERVICE=transfers yarn workspace @connext/cartographer-poller start
+SERVICE=transfers yarn workspace @connext/cartographer-poller start -->
+
+#### We need to follow certain steps to run the poller:
+
+- Open a terminal in the repository root and run the following command to start a PostgreSQL database server in a Docker container:
+
+  ```sh
+  docker run -p 5432:5432 -e POSTGRES_PASSWORD=qwerty -d postgres
+  ```
+
+- Make sure that you have a PostgreSQL server running on your local machine. If you don't, please follow the instructions provided in the README to start a PostgreSQL server in a Docker container. To connect to the database, use the following connection string:
```

Comment:
This would be just the previous step, right ?

Reply:
Yeah, make sense, Just added this so that people can make sure from dashboard if they have Postgres server running or not, in case of any error while using the prev command, Let me know your opinion on this @preethamr